.github/workflows/ci.yml

```diff
@@ -104,8 +104,6 @@ jobs:
     with:
       project-name: Component[OpenTelemetry.Exporter.Geneva]
       code-cov-name: Exporter.Geneva
-      os-list: '[ "ubuntu-24.04" ]' # Note: This may be switched to latest once ubuntu-latest has a kernel version >= 6.8.0-1014-azure
-      tfm-list: '[ "net8.0" ]' # Note: Should be able to remove this once the above is using ubuntu-latest
```

Comment:
There is couple other occurrences of `tfm-list` which should be updated in this file (extended by net9.0)

Reply:
I don't think we need to do that until projects actually have `net9.0` targets added.